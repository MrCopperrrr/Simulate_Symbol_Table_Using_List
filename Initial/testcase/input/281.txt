ASSIGN n 439118
ASSIGN y 712770
ASSIGN v c
INSERT y string
ASSIGN o p
ASSIGN x 441990
INSERT o number
INSERT z string
INSERT z number
INSERT y string
INSERT z number
INSERT m string
INSERT x string