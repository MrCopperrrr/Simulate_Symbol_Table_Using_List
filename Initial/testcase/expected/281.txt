Undeclared: ASSIGN n 439118